awspricing/offers.py

```diff
@@ -197,16 +197,19 @@ def ondemand_hourly(self,
                         license_model=None,          # type: Optional[str]
                         preinstalled_software=None,  # type: Optional[str]
                         region=None,                 # type: Optional[str]
+                        sku=None,                    # type: Optional[str]
                         ):
         # type: (...) -> float
-        sku = self.get_sku(
-            instance_type,
-            operating_system=operating_system,
-            tenancy=tenancy,
-            license_model=license_model,
-            preinstalled_software=preinstalled_software,
-            region=region
-        )
+
+        if sku is None:
```

Comment:
Would you be able to add validation that `sku` and the args that compute it are mutually exclusive?